test/unit/controllers/actions/AuthActionSpec.scala

```diff
@@ -25,27 +25,27 @@ import views.html.choice_page
 
 class AuthActionSpec extends ControllerSpec {
 
-  val choicePage = new choice_page(mainTemplate, minimalAppConfig)
-
-  val controller =
-    new ChoiceController(mockAuthAction, stubMessagesControllerComponents(), choicePage)(ec)
-
-  "Auth Action" should {
-
-    "return InsufficientEnrolments when EORI number is missing" in {
-      userWithoutEori()
-
-      val result = controller.displayPage(None)(getRequest())
-
-      intercept[InsufficientEnrolments](status(result))
-    }
-
-    "return NoExternalId when External Id is missing" in {
-      userWithoutExternalId()
-
-      val result = controller.displayPage(None)(getRequest())
-
-      intercept[NoExternalId](status(result))
-    }
-  }
+//  val choicePage = new choice_page(mainTemplate, minimalAppConfig)
+//
+//  val controller =
+//    new ChoiceController(mockAuthAction, stubMessagesControllerComponents(), choicePage)(ec)
+//
+//  "Auth Action" should {
+//
+//    "return InsufficientEnrolments when EORI number is missing" in {
+//      userWithoutEori()
+//
+//      val result = controller.displayPage(None)(getRequest())
+//
+//      intercept[InsufficientEnrolments](status(result))
+//    }
+//
+//    "return NoExternalId when External Id is missing" in {
+//      userWithoutExternalId()
+//
+//      val result = controller.displayPage(None)(getRequest())
+//
+//      intercept[NoExternalId](status(result))
+//    }
+//  }
```

Comment:
Commented spec, but it's a spike, so probably you know about it

Reply:
Ohhh no.. well it is a spike with the intention of being merged into `master`

Reply:
I need to fix it

Reply:
this has been addressed